Source/Routing and Faulting/Scripting/Tests/Unit/Routing and Faulting Validation Unit Tests/Assets/Validation Test.nivssdf

```diff
@@ -5665,6 +5677,24 @@
                      <Property Name="user.CD.Modules.5">
 								<DependentNode Path="Targets/Controller/Hardware/SLSC/SLSC Chassis/Modules/Unused" />
 							</Property>
+                     <Property Name="user.CD.Revisions.0">
```

Comment:
There something strange happening with indentation here. I think you have a mix of tabs and spaces.

Reply:
Fixed.